### ISSUE TITLE
How do you add a directory?

### ISSUE THREAD

**Issue by ezhang7423** [kevin-hanselman/dud#153]
```
ubuntu@base ~/d/runs (main) [1]> dud stage add 2023-07-24 
Error: 2023-07-24: yaml: input error: read 2023-07-24: is a directory
```

If this isn't supported, it seems like a pretty essential feature...

**Comment by thorstenwagner** [comment]
Mh, it works different: 

1. Add your directory to stage file
```
dud stage gen -o my/dir/ | tee my_dir.yaml 
```

2. Add stage

```
dud stage add my_dir.yaml 
```

Do the rest:
```
dud commit
dud push
git add .
git commit -m "Added dir"
git push

```

**Comment by kevin-hanselman** [comment]
Thanks for the question, @ezhang7423! Sorry for the late response; I was away on vacation for a while.

@thorstenwagner has shown how this is done in Dud; thanks a bunch, @thorstenwagner!

TLDR: `stage gen` generates/creates a stage, and `stage add` adds a stage _file_ to Dud's index.
